source/reference/logs.txt

```diff
@@ -0,0 +1,15 @@
+=========================
+Application Log Reference
+=========================
+
+.. toctree::
+   :titlesonly:
+
+   Authentication Logs </reference/logs/authentication>
+   Change Stream Logs </reference/logs/changestream>
+   Endpoint Logs </reference/logs/endpoint>
+   Function Logs </reference/logs/function>
+   Service Logs </reference/logs/service>
+   Trigger Logs </reference/logs/trigger>
+   Schema Logs </reference/logs/schema>
+   Sync Logs </reference/logs/sync>
```

Comment:
shouuld we put links to all the pages under this? noticed you did for the other landing pages?

Reply:
Oh yeah good catch I meant to do this. I was testing to see if the toctree still outputs a table on its own but, alas, no.

Reply:
merp